homeworks/ekaterina.sharova_rogerosha/PokemonProject/pokemons.css

```diff
@@ -0,0 +1,152 @@
+* {
```

Comment:
Please move styles to folder styles and images to another folder

Reply:
As I understood, it must be in this file, because it doesn't want to work in another one

Reply:
It will work, if you correct the path to the css file
<img width="1154" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/73c52804-fc78-4ec7-a6e6-8bb127acfea9">

Reply:
Oh, I understood it in the wrong way firstly. Thank you!